<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear_Map"/>
    <addaction name="actionNew_Map"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <widget class="QMenu" name="menuFortune_s_Algorithm">
     <property name="title">
      <string>Fortune's Algorithm</string>
     </property>
     <addaction name="actionToggle_T"/>
     <addaction name="actionStep_N"/>
    </widget>
    <addaction name="menuFortune_s_Algorithm"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClear_Map">
   <property name="text">
    <string>Clear Map</string>
   </property>
  </action>
  <action name="actionNew_Map">
   <property name="text">
    <string>New Map</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionToggle_T">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toggle (T)</string>
   </property>
  </action>
  <action name="actionStep_N">
   <property name="text">
    <string>Step (N)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
